### ISSUE TITLE
Add the possibility to set the FilterIndex in the FileDialogSettings

### ISSUE THREAD

**Issue by gigios** [FantasticFiasco/mvvm-dialogs#25]
I have the necessity to select a different index of the filter instead the first.
For example I have this filter: "**Bitmap file (*.bmp)|*.bmp|PNG file (*.png)|*.png|All Files (\*.\*)|\*.\***" and I want to select the PNG as starting index.
The SaveFileDialog (and OpenFileDialog) inherits from FileDialog that allow to select a different index via FilterIndex.
A very easy solution is to add the FilterIndex property to the 'base class' *FileDialogSettings.cs+ in this way:
```c#
        /// <summary>
        /// Gets or sets the index of the filter currently selected in a file dialog.
        /// </summary>
        /// <returns>The <see cref="T:System.Int32" /> that is the index of the selected filter. The default is 1.</returns>
        public int FilterIndex { get; set; } = 1;
```
and after to set the property in the constructor of the wrapper classes (OpenFileDialogWrapper/SaveFileDialogWrapper).
The default value used is the same of the Win32 FilterIndex (for compatibility is base 1 instead base 0).
See [msdn: FileDialog.FilterIndex Property](https://msdn.microsoft.com/en-us/library/microsoft.win32.filedialog.filterindex(v=vs.110).aspx)

**Comment by FantasticFiasco** [comment]
Hi gigios and welcome to this repo!

Adding a filter index seems like the correct action, I must have missed that one when mapping the properties of the file browser dialog.

Is this something you would like to contribute? Otherwise I'll get right on it and in the upcoming days have a beta for you to try out.

**Comment by gigios** [comment]
For me it's not a problem to contribute with the project. The only problem is that I'm not an expert of GitHub, Cloned projects and Pull request :)
I will try today to do the fix.

**Comment by FantasticFiasco** [comment]
No worries, we'll do it together then.

The first thing you should do is to fork the project. When you've forked it, create a pull request (make sure the option to allow me push rights to the PR is selected), and then I'll try to help you out in any way I can.

**Comment by gigios** [comment]
Ok! Now I'm blocked.
I have forked the project, cloned locally, changed and updated on my repository (with a not necessary, probably, update also to the file .gitignore used to skip some ReSharper files).
Now I have pressed the "Pull Request" button and it's shown the "Comparing changes" where I'm able to see the tree changed files (plus the .gitignore).
Where is the setting described in your previous post? Is it after the click on "Create pull request"?
